ioctl/cmd/action/action.go

```diff
@@ -374,8 +366,12 @@ func Read(contract address.Address, amount *big.Int, bytecode []byte) (string, e
 	res, err := iotexapi.NewAPIServiceClient(conn).ReadContract(
 		ctx,
 		&iotexapi.ReadContractRequest{
-			Execution:     exec.Proto(),
-			CallerAddress: caller,
+			Execution: &iotextypes.Execution{
+				Amount:   amount,
+				Contract: contract.String(),
+				Data:     bytecode,
+			},
+			CallerAddress: address.ZeroAddress,
```

Comment:
can we set zero address here? When we read contract (e.g., ERC20 ```balannceOf``` method), we might need a caller address(msg.sender)?

Reply:
discussed offline